### DISCUSSION TITLE
After update to v4.3 buttons go other places.

### DISCUSSION THREAD

**Discussion by ranirahn** [Novik/ruTorrent#2652] [Q&A] [answered]
Installed v4.3 and noticed that when I clicked on "RSS Downloader" button it opened up "Add RSS Feed" instead of "RSS Manager" that this button used to open. Can get to "RSS Manager" only with right click on RSS feeds and selecting that from there. Is it only me or was it changed on purpose?

Edit:

I found this

theWebUI.showRSS = function()
 {
         if($('#rssl').children().length)
        	theWebUI.RSSManager();
        else
		theDialogManager.toggle("dlgAddRSS");
 }

So its normal function if I have no rss feeds. Do I understand that correctly?
The problem is I have 9 RSS Feeds. So for somereason this function dont see that I have already feeds so it wants to add some. I tried todelete and add some of these feeds but nothing changed so for some reason it does not see that I already have feeds and dont open the RSS Manager. I dont actually know do my filters still work or is this also broken.
Another thing that I also dont see is if I disable or enable feeds it does not change anything. Usually the icon changed if feed was disabled. Now its not change icon but right click menu tell me its disabled. Very weird stuff.

**Comment by ranirahn** [comment]
For temporary fix I replaced `theDialogManager.toggle("dlgAddRSS");` with `theWebUI.RSSManager();` so now it always opens up the RSSManager, but it does not explain why it does not see my feeds.

**Comment by ranirahn** [comment]
Can anyone help me figure out why RSS plugin does not work right for me? Feeds work, data is there. RSS Manager downloads wanted torrents, but for some reason ruTorrent think I don't have ANY RSS feeds and don't know if they are disabled or not so something does not see feeds correctly, but I can't find the problem. I updated from v4.2.10 to v4.3.

I just get one idea. Maybe because I skipped that v4.2.11, I am missing some fixes in the code for it. Will test this out right now and see if that fixes it.

Edit:

v4.2.11 - everything works like it should be. Back to v4.3 and its broken again.

Edit2:

Not sure if anyone even read this, but I think something is wrong with this one
`if($('#rssl').children().length)`
Older versions `#rssl` existed, but could not find it in version v4.3. Witch is probably the reason some of the RSS functions dont work but others do. So some places have replacement for it, but not this and few other things. I just dont know what is the replacement. Tried even use clean install and add RSS feed and it still dot open RSS Manager.

Edit3:
I actually cant belive that I am the only one who have this problem. Code changed so much that finding the missing link is to much for me. Thought about making bug report, but I cant figure out enough details what is actually broken so I give up on that one. I just hope I am not the only one with this problem and someone else notice it too who can actually understand what is broken.

**Comment by stickz** [comment]
Please file a bug report under the issues section.

**Comment by ranirahn** [comment, marked as answer]
Managed to fix my issues myself with pull request https://github.com/Novik/ruTorrent/pull/2656
